src/Raven.Pal/src/posix/spawn.c

```diff
@@ -28,21 +28,40 @@ _parse_cmd_line(char* line, char *filename)
 {
     int argc = 2;
     char** argv = malloc(argc);
+    if (argv == NULL)
+        return NULL;
     argv[0] = filename;
     argv[--argc] = NULL;    
     if (line == NULL || *line == '\0')
+    {
+        char **tmpargv = realloc(argv, sizeof(long)*(argc + 2));
+        if (tmpargv == NULL)
+        {
+            free(argv);
+            return NULL;
+        }
+        argv = tmpargv;        
+        argv[1] = "";
+        argv[2] = NULL;
         return argv;
+    }
     
     char *rest = line;
     char *token;
 
-    while((token = strtok_r(rest, " \t\n", &rest)))
-    {
+    const char delimiters[] = " \t\n";
+    while((token = strtok_r(rest, delimiters, &rest)))
```

Comment:
`rest` is shown here twice, pretty sure you need to specify different variable here.

Reply:
nop. I meant it to be this way.

Reply:
Although it works, you are right, the correct usage is:
https://github.com/ravendb/ravendb/pull/9723/commits/34cebfa76f1074867f2e96a9f7d089719cf23ddb